pandas/core/frame.py

```diff
@@ -894,6 +893,10 @@ def itertuples(self, index=True, name="Pandas"):
             The name of the returned namedtuples or None to return regular
```

Comment:
Can you change the types of the parameters, from `boolean` to `bool`, and from `string` to `str` please?

Reply:
Done!